tests/neural_compressor/test_optimization.py

```diff
@@ -489,9 +489,10 @@ def test_weight_only_quantization(self, methodology, bits):
                 batch_size=5,
                 seq_len=32,
                 block_size=16,
+                use_layer_wise=True,
             )
         else:
-            quantization_config = RtnConfig(bits=bits, group_size=8)
+            quantization_config = RtnConfig(bits=bits, group_size=8, use_layer_wise=True)
```

Comment:
Why do we need to specify it when creating the quantization config ? Looks like with the current integration this information will be ignored (`load_empty_model` will be called in all cases)